README.md

```diff
@@ -416,6 +416,10 @@ If the subscriber has already been aborted (i.e., `subscriber.signal.aborted` is
 
 We propose the following operators in addition to the `Observable` interface:
 
+- `catch()`
+  - Like `Promise.catch()`, it takes a callback which gets fired after the source
```

Comment:
```suggestion
  - Like `Promise#catch()`, it takes a callback which gets fired after the source
```

Super nit: just since this is not a static method I want to use the weird `#` syntax to indicate that you can't actually open up DevTools and just call `Promise.catch()` like the example here makes it look like.